CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### citus v10.1.1 (August 5, 2021) ###
+
+* Fixes master_update_table_statistics UDF
```

Comment:
Improves master_update_table_statistics and provides distributed deadlock detection

Reply:
Is this udf public ? If not, maybe we can say something like:

Improves the logic/infra(?) that detects distributed deadlocks when updating distributed table statistics

Reply:
it is public yes

Reply:
I am not aware of a doc but as you said it is used in cloud-plane and I thought that was enough for it to be considered public :)

I mean users can use it, if users couldn't use it I think we wouldn't need to add even an entry because the change wouldn't affect the users in any way. 

I kind of think the "cloud-plane" as a user as well.

Reply:
I wonder what @naisila thinks about this alternative changelog entries